docs/section4/4-3.md

```diff
@@ -0,0 +1,46 @@
+# ASHVAC – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** HVAC Fan Schedules. Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. Not applicable for for heating or cooling system is being simulated only to meet the requirements described in Table G3.1.
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
```

Comment:
I think this should be each HeatingVentilationAirAconditioningSystem

Reply:
Okay, change made.

Reply:
@claperle I dont see the change. I'm assuming the intent was to change 'Evaluate each data element ' to 'Evaluate each HeatingVentilationAirAconditioningSystem'. Maybe the changes didn't save?

Reply:
@supriyagoel I just did not quite under, this has been updated